src/draft-yang-tls-tls13-sm-suites-01.md

```diff
@@ -411,33 +411,52 @@ associated data to that mode of operation. An authentication tag conformed to
 what Section 5.2 of TLSv1.3 {{RFC8446}} requires is used, which in details SHOULD
 be constructed by the TLS record header. The AEAD_SM4_GCM ciphertext is formed by
 appending the authentication tag provided as an output to the GCM encryption
-operation to the ciphertext that is output by that operation. The input and
-output lengths are as follows:
+operation to the ciphertext that is output by that operation. AES-AEAD_SM4_GCM has 
+four inputs: a SM4 key, an initialization vector (IV), a plaintext content, and optional 
```

Comment:
*an* SM4 key

Reply:
i think should be 'a'，just(a,e,i,o,u) need to be 'an':-)

Reply:
No. `S` pronounces as something like `eh-ss`, so it is an `e`. This should depend on how the word pronounces instead of how the word is written.

Reply:
The same thing happens that you should say (or write) 'an RFC' instead of 'a RFC'...